core/mondoo-kubernetes-inventory.mql.yaml

```diff
@@ -1,6 +1,7 @@
 packs:
```

Comment:
This file is missing a version for the different *Set inventories at the end.

Reply:
Thank you, fixed it